com.github.zadam.trilium.yml

```diff
@@ -0,0 +1,97 @@
+app-id: com.github.zadam.trilium
+
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+
+command: trilium
+
+separate-locales: false
+
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home
```

Comment:
does it really need access to the whole home directory?

Reply:
The app has user-configurable directory where it stores its data. By default it looks for data in `~/.local/share/trilium-data` and `~/trilium-data` but again, allows to store them elsewhere (configurable via env. variable). So I'm not sure how to approach this.